src/Components/profile/index.js

```diff
@@ -31,24 +32,34 @@ const Profile = () => {
   const [goToDetailEvent, setGoToDetailEvent] = useState(false);
   const [goToEditEvent, setGoToEditEvent] = useState(false);
   const [eventToEdit, setEventToEdit] = useState("");
-  const [eventsList, setEventList] = useState(
-    JSON.parse(localStorage.getItem("Events"))
-  );
+  const [eventsList, setEventList] = useState(getEventsFromLocalStorage());
   const [pageNumber, setPageNumber] = useState(0);
+  const userInformation = getFromLocalStorage();
+  const userId = userInformation.idUser;
   const eventsPerPage = 6;
+
   const pageCount =
     eventsList && eventsList.length
       ? Math.ceil(eventsList.length / eventsPerPage)
       : 0;
+
   const pagesVisited = pageNumber * eventsPerPage;
+
   let currentEvents =
     eventsList && eventsList.length
-      ? eventsList.slice(pagesVisited, pagesVisited + eventsPerPage)
+      ? eventsList
+          .slice(pagesVisited, pagesVisited + eventsPerPage)
+          .map((event) => {
+            if (event.users.includes(userId)) {
```

Comment:
lets move this logic inside `getEventsFromLocalStorage`, the idea is to filter the events inside this function, and return the events ready